src/templates/en/2019/contributors.html

```diff
@@ -107,20 +171,21 @@ <h1>
     <div class="contributor-grid">
     {% for id, contributor in config.contributors.items() %}
       <div class="contributor {{ ' '.join(contributor.teams) }}">
-        <img class="contributor-avatar" src="{{ contributor.avatar_url }}" height="100" width="100" alt="{{ contributor.name }} avatar">
```

Comment:
`avatar_url` is a pre-generated URL that falls back to a random character illustration from the Almanac design. We should try to keep that functionality.